### ISSUE TITLE
[Ubuntu] 20.04 LTS cannot install

### ISSUE THREAD

**Issue by Alice52** [flameshot-org/flameshot#1642]
**Flameshot version**

- 0.9.0: `flameshot-0.9.0-1.ubuntu-20.04.amd64.deb`

**Describe the bug**

- ocur exception when install, and I cannot upgrate `libqt5core5a`
```shell
# os info
➜  software cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.1 LTS"

# occur error when install flameshot
➜ sudo dpkg -i flameshot-0.9.0-1.ubuntu-20.04.amd64.deb
Selecting previously unselected package flameshot.
(Reading database ... 197855 files and directories currently installed.)
Preparing to unpack flameshot-0.9.0-1.ubuntu-20.04.amd64.deb ...
Unpacking flameshot (0.9.0-1) ...
dpkg: dependency problems prevent configuration of flameshot:
 flameshot depends on libqt5svg5 (>= 5.9.0~); however:
  Package libqt5svg5 is not installed.
 flameshot depends on libqt5core5a (>= 5.12.2); however:
  Version of libqt5core5a:amd64 on system is 5.9.5+dfsg-0ubuntu2.5.
 flameshot depends on libqt5dbus5 (>= 5.9.0~); however:
  Package libqt5dbus5 is not installed.
 flameshot depends on libqt5gui5 (>= 5.10.0) | libqt5gui5-gles (>= 5.10.0); however:
  Package libqt5gui5 is not installed.
  Package libqt5gui5-gles is not installed.
 flameshot depends on libqt5network5 (>= 5.9.0~); however:
  Package libqt5network5 is not installed.
 flameshot depends on libqt5widgets5 (>= 5.11.0~rc1); however:
  Package libqt5widgets5 is not installed.

dpkg: error processing package flameshot (--install):
 dependency problems - leaving unconfigured
Processing triggers for gnome-menus (3.36.0-1ubuntu1) ...
Processing triggers for desktop-file-utils (0.24-1ubuntu3) ...
Processing triggers for mime-support (3.64ubuntu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Errors were encountered while processing:
 flameshot
```

**To Reproduce**

- N/A

**Expected behavior**

- can install and run successlly, please help me out this truble.

**System Information**

- Ubuntu 20.04.1 LTS
@lupoDharkael @borgmanJeremy @p4ul

**Comment by hosiet** [comment]
Please do NOT use `dpkg -i` to install `.deb` packages anymore. Using `dpkg` will not automatically solve dependencies and you have to solve it manually. This is strongly discouraged unless you know what you are doing.

The preferred way is to use `apt` to install local `.deb` packages. Whenever you want to use `dpkg -i` to install deb package, replace `dpkg` call with the following command:

```
sudo apt install ./flameshot-0.9.0-1.ubuntu-20.04.amd64.deb
```

Note that you must use a relative or absolute path for the targeted `deb` file. Running `sudo apt install flameshot-0.9.0-1.ubuntu-20.04.amd64.deb` will fail. Running `sudo apt install ./flameshot-0.9.0-1.ubuntu-20.04.amd64.deb` will finish successfully.

**Comment by Alice52** [comment]
@hosiet 
Thanks for reply, but it's still not work. And I can't upgrade these libs about qt. It's fastest version in this machine.

```shell
➜  sudo apt install ./flameshot-0.9.0-1.ubuntu-20.04.amd64.deb
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'flameshot' instead of './flameshot-0.9.0-1.ubuntu-20.04.amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 flameshot : Depends: libqt5core5a (>= 5.12.2) but 5.9.5+dfsg-0ubuntu2.5 is to be installed
             Depends: libqt5gui5 (>= 5.10.0) but 5.9.5+dfsg-0ubuntu2.5 is to be installed or
                      libqt5gui5-gles (>= 5.10.0) but it is not installable
             Depends: libqt5widgets5 (>= 5.11.0~rc1) but 5.9.5+dfsg-0ubuntu2.5 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by hosiet** [comment]
@Alice52 Ubuntu 20.04 LTS come with Qt 5.12. If you only have Qt 5.9 available, you are NOT using Ubuntu 20.04. Maybe you are using Ubuntu 18.04 but you wrongly assume that you are using Ubuntu 20.04, or you have wrongly configured the software repository to use repos for Ubuntu 18.04.

In either case, you need to fix your system setup first, and continue with flameshot installation after these system issues are solved.

**Comment by Alice52** [comment]
Thanks, Yeah, I wrongly configured the software repository, which is for Ubuntu 18.04.  It really save my days.
